### PR TITLE
!: add Chinese translation

### PR DESCRIPTION
**Pull request by Ren-Ma**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.zh/common/!.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+- 替换指定数量的行回查的命令：
+
+`!-{{number}}`
+
+- 替换最近开始以某个字符串的命令：
+
+`!{{string}}`
+
+- 替换最新命令的参数：
```
Comment (kbdharun) [inline pages.zh/common/!.md:22]:
```suggestion
- 替换最新命令的参数：

`{{命令}} !*`
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

- Please sign the CLA in the above comment for this PR to go through.
- A command is missing after the last description (I have added it in my suggestion).

---

**Note**: I am not a native speaker and these placeholder suggestions were done based on the existing `zh` pages we have here. For any additional queries, I would suggest asking the native speakers (code owners) instead.

File: pages.zh/common/!.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+- 使用`sudo`并替换为上一个命令来运行它：
+
+`sudo !!`
+
+- 根据通过`history`找到的行号替换命令：
+
+`!{{number}}`
+
+- 替换指定数量的行回查的命令：
```
Comment (einverne) [inline pages.zh/common/!.md:14]:
```suggestion
- 替换指定数量的行回查的命令：
```

What do you mean `指定数量的行回查的命令` here, I check the English description

```
Substitute with a command that was used a specified number of lines back:
```

I think you may be referring to `specified number of lines back`, but I never see any translation like `行回查命令`. The English description is much clearer than it. Shall we use another expression?

**Review by einverne — CHANGES_REQUESTED** [review]
Hi thank you for the translation page, I left a few suggestions for you.

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
@einverne Any updates on this?

**Review by einverne — APPROVED** [review]
LGTM.

**Pull request merged**